flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/runtime/operators/deduplicate/MiniBatchDeduplicateKeepLastRowFunctionTest.java

```diff
@@ -41,23 +41,28 @@
  */
 public class MiniBatchDeduplicateKeepLastRowFunctionTest extends DeduplicateFunctionTestBase {
 
-	private TypeSerializer<BaseRow> typeSerializer = inputRowType.createSerializer(new ExecutionConfig());
+	private TypeSerializer<BaseRow> typeSerializer = inputRowType
+		.createSerializer(new ExecutionConfig());
 
-	private MiniBatchDeduplicateKeepLastRowFunction createFunction(boolean generateRetraction) {
-		return new MiniBatchDeduplicateKeepLastRowFunction(inputRowType, generateRetraction, typeSerializer);
+	private MiniBatchDeduplicateKeepLastRowFunction createFunction(boolean generateRetraction,
+		long minRetentionTime) {
+		return new MiniBatchDeduplicateKeepLastRowFunction(inputRowType, generateRetraction,
+			typeSerializer, minRetentionTime);
 	}
 
 	private OneInputStreamOperatorTestHarness<BaseRow, BaseRow> createTestHarness(
-			MiniBatchDeduplicateKeepLastRowFunction func)
-			throws Exception {
+		MiniBatchDeduplicateKeepLastRowFunction func)
+		throws Exception {
 		CountBundleTrigger<Tuple2<String, String>> trigger = new CountBundleTrigger<>(3);
 		KeyedMapBundleOperator op = new KeyedMapBundleOperator(func, trigger);
-		return new KeyedOneInputStreamOperatorTestHarness<>(op, rowKeySelector, rowKeySelector.getProducedType());
+		return new KeyedOneInputStreamOperatorTestHarness<>(op, rowKeySelector,
+			rowKeySelector.getProducedType());
```

Comment:
Is there any changes between these lines? It's not encouraged to refactor code sytle among some other changes.  If it is for code style, the later one is not good: 
1) please add an additional indent for new line parameter.
2) please put each parameter in a speparate line if you think the method call is too long. 

Please refer a more detailed code style guideline: https://flink.apache.org/contributing/code-style-and-quality-preamble.html

Reply:
sorry, I didn't notice that